tensorboard/webapp/app_container.ng.html

```diff
@@ -19,3 +19,4 @@
 <plugins class="plugins"></plugins>
 <reloader></reloader>
 <hash-storage></hash-storage>
+<settings-polymer-interop></settings-polymer-interop>
```

Comment:
Is there some scenario when TensorBoard should not have `settings-polymer-interop`?  My understanding was that elements made available in this HTML template allow for customization, e.g. custom header.

If hash-storage and settings-polymer-interop are actually required functionality, it would be nice not to have to update each variant.  Wdyt about making some `<core-features></core-features>` that internally appends required fields hash-storage (which already lives in core/), settings-polymer-interop?  (not necessarily in this CL)

Reply:
I don't feel too strongly about it but I am inclined _not to_ create `core-features`. If there is one component that you do not want, you will end up needing to fork it and copy-pasta anyways.

Reply:
I'm fine with landing as-is for now, since there are only a handful of elements in this template.

I still prefer fewer points of customization, since adding support for more (when there is a use case) is always in our control.

If this template had 50 elements that customized some TB feature, we must remember to maintain configurations of all variants.  That requires relying on memory, rather than process.